lib/canvas/live_events.rb

```diff
@@ -184,6 +205,23 @@ def self.wiki_page_deleted(page)
     })
   end
 
+  def self.attachment_created(attachment)
+    post_event_stringified('attachment_created', get_attachment_data(attachment))
+  end
+
+  def self.attachment_updated(attachment, old_display_name)
+    payload = get_attachment_data(attachment)
+    if old_display_name
```

Comment:
We do want to cherry-pick at this point. We're starting conservatively. Down the road, we may want to look at just sending the `changes` hash.